app/controllers/submissions_controller.rb

```diff
@@ -27,6 +27,10 @@ def set_submission
   end
 
   def submission_params
-    params.expect(submission: [:abstract])
+    params.expect(submission: %i[abstract sulicense cclicense embargo]).merge(cclicensetype:)
+  end
+
+  def cclicensetype
```

Comment:
Is `cclicensetype` a carry-over name we want to stick with, or should we consider a more idiomatic name?